scripts/xtensa-build-zephyr.py

```diff
@@ -443,27 +444,26 @@ def west_update():
 	execute_command(["west", "update"], check=True, timeout=3000, cwd=west_top)
 
 
-def get_sof_version(abs_build_dir):
+def get_sof_version():
 	"""[summary] Get version string major.minor.micro of SOF
 	firmware file. When building multiple platforms from the same SOF
 	commit, all platforms share the same version. So for the 1st platform,
-	generate the version string from sof_version.h and later platforms will
+	extract the version information from sof/versions.json and later platforms will
 	reuse it.
 	"""
 	global sof_fw_version
 	if sof_fw_version:
 		return sof_fw_version
 
 	versions = {}
-	with open(pathlib.Path(abs_build_dir,
-		  "zephyr/include/generated/sof_versions.h"), encoding="utf8") as hfile:
-		for hline in hfile:
-			words = hline.split()
-			if words[0] == '#define':
-				versions[words[1]] = words[2]
-	sof_fw_version = versions['SOF_MAJOR'] + '.' + versions['SOF_MINOR'] + '.' + \
-		      versions['SOF_MICRO']
-
+	with open(SOF_TOP / "versions.json") as versions_file:
+		versions = json.load(versions_file)
+		sof_micro = versions['SOF'].get('MICRO')
```

Comment:
The get here allow a default value if the value we want to get doesn't exist, why not just simply

`sof_micro = versions[''SOF].get('MICRO', '0')`

Reply:
Done